From e848345f3c15528c59297d35185f2189931260c7 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Tue, 15 May 2018 14:36:08 +0800
Subject: [PATCH 027/129] media: intel-ipu4: imx319: Enable ACPI registration

Message for Open Source:
Enable ACPI registration

Message for Internal:
[Issue/Feature] ACPI is not enabled.
[Root Cause/Changes] Enable ACPI registration

Change-Id: Ib52bb53f3ae9930780ccd351b77bca965634c0f4
Tracked-On: #JICCCN-171
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx319.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/imx319.c b/drivers/media/i2c/imx319.c
index 0821e00..2a35325 100644
--- a/drivers/media/i2c/imx319.c
+++ b/drivers/media/i2c/imx319.c
@@ -2498,7 +2498,7 @@ static int imx319_remove(struct i2c_client *client)
 
 #ifdef CONFIG_ACPI
 static const struct acpi_device_id imx319_acpi_ids[] = {
-	{ "IMX319" },
+	{ "SONY319A" },
 	{ /* sentinel */ }
 };
 MODULE_DEVICE_TABLE(acpi, imx319_acpi_ids);
@@ -2509,7 +2509,9 @@ static int imx319_remove(struct i2c_client *client)
 		.name = "imx319",
 		.owner = THIS_MODULE,
 		.pm = &imx319_pm_ops,
-		/*.acpi_match_table = ACPI_PTR(imx319_acpi_ids),*/
+#ifdef CONFIG_ACPI
+		.acpi_match_table = ACPI_PTR(imx319_acpi_ids),
+#endif
 	},
 	.probe = imx319_probe,
 	.remove = imx319_remove,
-- 
1.9.1

